.travis.yml

```diff
@@ -13,25 +13,32 @@
 # limitations under the License.
 
 script:
-  - BALLERINA_VERSION=${VERSION:-1.0.0}
-  - echo -e "\033[36m${BALLERINA_VERSION}\033[m"
+  # - BALLERINA_VERSION=${VERSION:-1.0.0}
+  # - echo -e "\033[36m${BALLERINA_VERSION}\033[m"
 
-  - DISTRIBUTION_NAME=ballerina-${BALLERINA_VERSION}
-  - echo -e "\033[36m${DISTRIBUTION_NAME}\033[m"
+  # - DISTRIBUTION_NAME=ballerina-${BALLERINA_VERSION}
+  # - echo -e "\033[36m${DISTRIBUTION_NAME}\033[m"
 
-  - DOWNLOAD_CHANNEL=${CHANNEL:-nightly}
-  - echo -e "\033[36m${DOWNLOAD_CHANNEL}\033[m"
+  # - DOWNLOAD_CHANNEL=${CHANNEL:-nightly}
+  # - echo -e "\033[36m${DOWNLOAD_CHANNEL}\033[m"
 
-  - DOWNLOAD_LOCATION=${URL:-https://product-dist.ballerina.io}
-  - echo -e "\033[36m${DOWNLOAD_LOCATION}\033[m"
+  # - DOWNLOAD_LOCATION=${URL:-https://product-dist.ballerina.io}
+  # - echo -e "\033[36m${DOWNLOAD_LOCATION}\033[m"
 
-  - echo -e "\033[36m${DOWNLOAD_LOCATION}/${DOWNLOAD_CHANNEL}/${BALLERINA_VERSION}${RC:-}/${DISTRIBUTION_NAME}${RC:-}.zip\033[m"
-  - wget ${DOWNLOAD_LOCATION}/${DOWNLOAD_CHANNEL}/${BALLERINA_VERSION}${RC:-}/${DISTRIBUTION_NAME}${RC:-}.zip
+  # - echo -e "\033[36m${DOWNLOAD_LOCATION}/${DOWNLOAD_CHANNEL}/${BALLERINA_VERSION}${RC:-}/${DISTRIBUTION_NAME}${RC:-}.zip\033[m"
+  # - wget ${DOWNLOAD_LOCATION}/${DOWNLOAD_CHANNEL}/${BALLERINA_VERSION}${RC:-}/${DISTRIBUTION_NAME}${RC:-}.zip
 
-  - unzip -q ${DISTRIBUTION_NAME}${RC:-}.zip
+  # - unzip -q ${DISTRIBUTION_NAME}${RC:-}.zip
 
-  - export PATH=${PATH}:$(pwd)/${DISTRIBUTION_NAME}/bin
+  # - export PATH=${PATH}:$(pwd)/${DISTRIBUTION_NAME}/bin
 
-  - ballerina --version
+  # - ballerina --version
+
+  # - ballerina build --skip-tests amazonsqs
 
-  - ballerina build --skip-tests amazonsqs
\ No newline at end of file
+
+  - wget https://product-dist.ballerina.io/downloads/1.0.0/ballerina-linux-installer-x64-1.0.0.deb
```

Comment:
Can't we use the parameterized variables for this?

Reply:
We have not yet parameterized variables as Ballerina team has not confirmed the hosted location and name format of the distribution package.